projects/netgrif-components-core/src/lib/panel/task-panel/abstract-task-panel.component.ts

```diff
@@ -46,6 +46,10 @@ import {PermissionService} from '../../authorization/permission/permission.servi
 import {ChangedFieldsService} from '../../changed-fields/services/changed-fields.service';
 import {ChangedFieldsMap} from '../../event/services/interfaces/changed-fields-map';
 
+export interface Context {
+    [k: string]: (str?: string) => boolean | string | void;
```

Comment:
the context does not provide any method with this signature. It provides a set of 17 methods and nothing else. The interface should reflect that.

`Context` is also a very generic name and does not reflect what the interface represents

Reply:
Done